bin/epaa.py

```diff
@@ -808,7 +808,7 @@ def make_predictions_from_variants(variants_all, methods, alleles, minlength, ma
         df.reset_index(inplace=True)
 
         for c in df.columns:
-            if '*' in str(c):
+            if ('HLA-' in str(c)) or ('H-2-' in str(c)):
```

Comment:
Can we be sure that the allele name in the column contains the `HLA-` prefix? I am not sure right now.

Reply:
This is how it is currently done in `FRED2`: 

```
if name.startswith("H2-"):
     return MouseAllele(name, prob=prob)
if name.count("*") > 1:
     return CombinedAllele(name, prob=prob)
```

Reply:
Hi @christopher-mohr ,
I think we can be sure. The `result` returned by `predict()` contains the allele as type `Allele`. (In `ANN.py` returned from `revert_allele_repr()` and in `External.py`, `PSSM.py`, `SVM.py` it is checked that the allele is an instance of type `Allele`.) When converting this to a `str` (in `epaa.py`) it should `return '%s-%s*%s:%s' % (str(self.organism), str(self.locus), str(self.supertype), str(self.subtype))` and thus contain `HLA-`.